<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QFRDRImagingFCSCountrateDisplay</class>
 <widget class="QWidget" name="QFRDRImagingFCSCountrateDisplay">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>968</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QVisibleHandleSplitter" name="splitterH">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout5" stretch="0,1,0">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;&amp;amp;image/video display:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="buddy">
            <cstring>cmbImage</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cmbImage">
           <property name="toolTip">
            <string>select the image you want to display below</string>
           </property>
           <property name="sizeAdjustPolicy">
            <enum>QComboBox::AdjustToContents</enum>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="JKQtPlotter" name="pltImage" native="true">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFPLayerControls" name="player" native="true">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_2"/>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QVisibleHandleSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="layoutWidget1">
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;&amp;amp;timetrace/statistics display:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="buddy">
             <cstring>cmbGraph</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cmbGraph">
            <property name="toolTip">
             <string>select the count rate display mode you want to see below</string>
            </property>
            <property name="sizeAdjustPolicy">
             <enum>QComboBox::AdjustToContents</enum>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="JKQtPlotter" name="pltIntensity" native="true">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>100</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="widBleach" native="true">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout_4" rowstretch="0,1,1">
        <property name="margin">
         <number>0</number>
        </property>
        <item row="1" column="1" rowspan="2">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="btnFit">
            <property name="text">
             <string>&amp;Fit</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="1" column="0" rowspan="2">
         <widget class="QTableView" name="tableView"/>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="labIntensityInfo">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>JKQtPlotter</class>
   <extends>QWidget</extends>
   <header>jkqtplotter.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QVisibleHandleSplitter</class>
   <extends>QSplitter</extends>
   <header>qvisiblehandlesplitter.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QFPLayerControls</class>
   <extends>QWidget</extends>
   <header>qfplayercontrols.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cmbImage</tabstop>
  <tabstop>cmbGraph</tabstop>
  <tabstop>tableView</tabstop>
  <tabstop>btnFit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cmbImage</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>QFRDRImagingFCSCountrateDisplay</receiver>
   <slot>displayImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnFit</sender>
   <signal>clicked()</signal>
   <receiver>QFRDRImagingFCSCountrateDisplay</receiver>
   <slot>calcFit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>966</x>
     <y>258</y>
    </hint>
    <hint type="destinationlabel">
     <x>961</x>
     <y>430</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmbGraph</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>QFRDRImagingFCSCountrateDisplay</receiver>
   <slot>displayData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>276</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>-6</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>calcFit()</slot>
  <slot>displayImage()</slot>
  <slot>displayData()</slot>
 </slots>
</ui>
